engine/apps/schedules/migrations/0006_customoncallshift_rotation_start.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.2.13 on 2022-07-12 08:03
+
+from django.db import migrations, models
+
+
+def fill_rotation_start_field(apps, schema_editor):
+    CustomOnCallShift = apps.get_model("schedules", "CustomOnCallShift")
+    shifts = CustomOnCallShift.objects.all()
+    shifts_to_update = []
+    for shift in shifts:
+        shift.rotation_start = shift.start
+        shifts_to_update.append(shift)
+
+    CustomOnCallShift.objects.bulk_update(shifts_to_update, ["rotation_start"], batch_size=5000)
```

Comment:
I think using update + [F expression ](https://docs.djangoproject.com/en/4.0/ref/models/expressions/#f-expressions)should be more efficient in this case (although we don't have too many rows):
```
from django.db.models import F
CustomOnCallShift.objects.update(rotation_start=F('start'))
```
(it should be something like that, didn't try it)